action.yaml

```diff
@@ -27,6 +27,10 @@ inputs:
     default: "openvex"
   custom-socket:
     description: "Custom socket address if setting up containerd image store"
+  max_retries:
+    description: "Max retries on timeout error"
```

Comment:
you will also need to pass in the `max_retries` argument into the `copa-action` container we run in `action.yaml`. This is what runs the `entrypoint.sh` code you are modifying to do the retries.

Reply:
@ashnamehrotra Yeah, missed that :sweat_smile:.
Thank you for the fast review :smile:.